### DISCUSSION TITLE
weblcd throwing an error, setup hasnt been changed

### DISCUSSION THREAD

**Discussion by srolock** [artisan-roaster-scope/artisan#1258] [Q&A]
running 2.8.4, cannot get weblcd to enable, click the box and the following error is noted

1 11:38:26.181 Exception: startWebLCDs() 'NoneType' object has no attribute 'write'@line 17652

I havent changed anything in the last month 0r so but recently the weblcd isnt started nor is reachable from remote pc.

Any idea is appreciated

**Comment by MAKOMO** [comment]
The error message is misleading. Artisan could not start its internal WebServer on the given port (8080 by default). Either another service is using it or a hanging Artisan WebServer is still connected to it. You can set a different port, like 8087, or kill/stop the hanging process blocking the port 8080.

**Reply by srolock** [reply]
changed port, reboot pc , No change
netstat has nothing using 8080, every time I change it it throws the same error.
No firewalls on, and artisan is about the only thing on that PC
Win 11 Pro

**Reply by MAKOMO** [reply]
Strange. It worked before. You did not change a thing and now it does not work!? Did you activate a firewall? Did you try using a different port?

**Reply by srolock** [reply]
Nothing I could find with firewalls or blocking or ports in use etc.
BUT I installed continuous build 2.8.5 and problem is gone. All is good with the world!
Didnt explore any new items in 2.8.5 but it corrected the web led issues for me and didnt hurt anything else I had setup.

Thanks,

Steve
